app/src/Manager/AdvisorManager.php

```diff
@@ -82,4 +76,23 @@ private function getAdviceFromResponseBody(\stdClass $body): array
             ],
         ];
     }
+
+    private function getBodyFromGame(Game $game): string
+    {
+        $body = [
```

Comment:
Maybe you can write like that:

```php
 $body = [
            'Grid' => $game->getBoard(),
            'Player' => $game->getCurrentPlayer(),
           'SelectedCube' =>  [
                'Coords' => ['X' => -1, 'Y' => -1],
                'Value' => -1,
                 ];
        ];

        $selectedCube = $game->getSelectedCube();
        if $selectedCube {
          $body['SelectedCube'] = [
              'Coords' => ['X' => $selectedCube['x'] , 'Y' => $selectedCube['y'] ],
              'Value' =>  $game->getBoard()[$cubeX][$cubeY,
          ];
      }
```

No need to change in this PR, it's just advice